js/bundledPlugins/previouslyUsedKeywords.js

```diff
@@ -0,0 +1,86 @@
+var AssessmentResult = require( "../values/AssessmentResult.js" );
+var isUndefined = require( "lodash/isUndefined" );
+
+/**
+ * Constructs the PreviouslyUsedKeywords plugin
```

Comment:
This documentation is redundant. We should document what you can do with a PreviouslyUsedKeywords plugin.